lib/ecto/query/builder.ex

```diff
@@ -341,36 +341,9 @@ defmodule Ecto.Query.Builder do
   end
 
   # in operator
-  def escape({:in, _, [left, right]} = expr, type, params_acc, vars, env)
-      when is_list(right)
-      when is_tuple(right) and elem(right, 0) in ~w(sigil_w sigil_W)a do
```

Comment:
I wonder if the simplest fix is to add ~w(sigil_w sigil_W @) to this line. Then, if necessary, also add a Macro.expand_once in the body?

Reply:
Good call. That should work because the other function head is meant for fragment, Query.API stuff like type and subquery. So they won't be put into a module attribute.

Changed!